controller.go

```diff
@@ -69,6 +70,8 @@ func loadSettings() error {
 		"sets the health check path for the created target groups")
 	flag.UintVar(&healthCheckPort, "health-check-port", aws.DefaultHealthCheckPort,
 		"sets the health check port for the created target groups")
+	flag.UintVar(&upstreamPort, "upstream-port", 0,
```

Comment:
By doing that it will break backward compatibility. Is that ok?

Reply:
I see!

Actually I would rather break backwards compatibility here instead of carrying over this weird assumption that healthcheckport -> port.

@szuecs what do you think about it?

Reply:
I am in for breaking backwards compatibility, but please add a note in the readme similar to the ones we already have to explain the changes and what you have to do to update and might need to clean up.
Thanks for your great work!